### ISSUE TITLE
Create database with multi_select doesn't work

### ISSUE THREAD

**Issue by landicefu** [seratch/notion-sdk-jvm#67]
This is my code, and if I remove the Tags column then it works.
```
    client.createDatabase(
            parent = DatabaseParent.page(page.id),
            title = listOf(DatabaseProperty.RichText(text = DatabaseProperty.RichText.Text(content = folderName), plainText = folderName)),
            isInline = true,
            properties = mapOf(
                "Name" to TitlePropertySchema(),
                "isFolder" to CheckboxPropertySchema(),
                "Tags" to MultiSelectPropertySchema(), // this cause the problem
                "Mime" to RichTextPropertySchema(),
                "File" to FilePropertySchema(),
                "Size In Bytes" to NumberPropertySchema(),
                "Upload Complete" to CheckboxPropertySchema(),
                "Thumbnail" to FilePropertySchema(),
            )
        )
````

The error response is:
```
body.properties.Tags.people should be defined, instead was `undefined`.
    body.properties.Tags.files should be defined, instead was `undefined`.
    body.properties.Tags.email should be defined, instead was `undefined`.
    body.properties.Tags.phone_number should be defined, instead was `undefined`.
    body.properties.Tags.date should be defined, instead was `undefined`.
    body.properties.Tags.checkbox should be defined, instead was `undefined`.
    body.properties.Tags.created_by should be defined, instead was `undefined`.
    body.properties.Tags.created_time should be defined, instead was `undefined`.
    body.properties.Tags.last_edited_by should be defined, instead was `undefined`.
    body.properties.Tags.last_edited_time should be defined, instead was `undefined`.
```

**Comment by landicefu** [comment]
I found other people reported the same issue, so I will just close this one

**Comment by seratch** [comment]
Hi @landicefu, thanks for reporting this issue. We've received a few feedback with the latest v1.5 series. Perhaps, the latest code may have some issues with the current Notion API version. If v1.4 works for you, please consider downgrading to v1.4 for now. Since I don't have the bandwidth to work on this SDK project recently, resolving the remaining issues may take a bit long.

**Comment by landicefu** [comment]
@seratch san, thanks for replying. I tried it and I doesn't work too, maybe they've changed some api spec again. I will take a look at the API document and if I can fix it I will just send a PR to you.

**Comment by seratch** [comment]
Fixed by #68
